src/supplemental/http/http_server.c

```diff
@@ -1496,6 +1496,8 @@ http_handle_dir(nni_aio *aio)
 
 	if (((rv = nni_http_res_alloc(&res)) != 0) ||
 	    ((rv = nni_http_res_set_status(res, NNG_HTTP_STATUS_OK)) != 0) ||
+	    ((rv = nni_http_res_set_header(
+	          res, "Access-Control-Allow-Origin", "*")) != 0) ||
```

Comment:
I think you probably already know this, but this is not a reasonable default.  I'm leaving this as a review, so that others can see why I'm not merging this this.